client/src/assets/lotties/heart-burst.json

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
If you use lottie files, please attribute the author.
https://lottiefiles.com/page/license

As long as this app is not available for end users, I would add a LICENSES file or add a comment to the top of this file.
When you release this app, consider to add a licenses section in your app.

Reply:
Fixed in commit https://github.com/ncbecker/podium/pull/42/commits/65f9303d68254705fab3b1d778f57e0ab920586f
Thanks 👍